packages/base/src/services.ts

```diff
@@ -8,6 +8,8 @@ export type EthqlServiceDefinition<Config, Service> = {
   };
 };
 
+export interface EthqlServiceDefinitions {}
```

Comment:
This interface is extended in every service but had no "empty" definition. This caused problem compiling `@ethql/base` as there is just one service and it's not imported anywhere in itself (it's imported in `ethql` package, but since we need to be able to build `@ethql/base` in isolation, this is not relevant here)/

Reply:
👍